build.sbt

```diff
@@ -4,15 +4,28 @@ version := "0.1"
 
 scalaVersion := "2.11.12"
 
+
 sparkVersion := "2.2.2"
 sparkComponents ++= Seq("sql")
 
-libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2" % "provided"
+libraryDependencies += "org.apache.spark" %% "spark-streaming" % "2.2.2"
+
+libraryDependencies += "org.apache.spark" %% "spark-sql" % "2.2.2"
 
 // https://mvnrepository.com/artifact/org.apache.spark/spark-streaming-twitter
-libraryDependencies += "org.apache.spark" %% "spark-streaming-twitter" % "1.6.3"
+libraryDependencies += "org.apache.bahir" %% "spark-streaming-twitter" % "2.2.1"
+
+
+
+/*
+val sparkVersion = "2.2.0"
```

Comment:
if you decide to comment out a piece of code and keep it in this disabled form in codebase, it implies that the code is important for some reason, so we could not just remove it. In this case, a text comment explaining a reason of preserving the code should be provided, so it's possible to understand the intention.

Reply:
Removed..